pkg/cmd/admin/project/new_project.go

```diff
@@ -83,6 +85,7 @@ func (o *NewProjectOptions) Run() error {
 	project.Annotations = make(map[string]string)
 	project.Annotations["description"] = o.Description
 	project.Annotations["displayName"] = o.DisplayName
+	project.Annotations["nodeSelector"] = o.NodeSelector
```

Comment:
so the order has come from up high that we need to namespace our annotation keys and not use camel casing.

so while you are consistent with what is there now, we need to change:

```
openshift.io/node-selector
```

As for the rationale of - instead of camelCase, it was an arbitrary choice to be consistent with labels.